.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "riscv-binutils-gdb"]
+	path = riscv-binutils-gdb
+	url = https://github.com/riscv/riscv-binutils-gdb.git
```

Comment:
Do you mean something like

````
  url = ../riscv-binutils-gdb
````

?  I don't like that, as it'll require people to clone something additional.